pkg/controller/kfservice/kfservice_controller_test.go

```diff
@@ -43,7 +43,7 @@ import (
 
 var c client.Client
 
-const timeout = time.Second * 10
+const timeout = time.Second * 30
```

Comment:
is it always failing in `DeleteCanary` test ?

Reply:
Seems that's failed for cases `TestCanaryReconcile`.

Reply:
Discussed with Dan, rollback the change.